Gemfile

```diff
@@ -27,4 +27,6 @@ gem 'puppet-lint-variable_contains_upcase'
 # rspec must be v2 for ruby 1.8.7
 if RUBY_VERSION >= '1.8.7' and RUBY_VERSION < '1.9'
   gem 'rspec', '~> 2.0'
+  # rake must be v10 for ruby 1.8.7
```

Comment:
You can revert this as it is now upstream and a potential source of the conflict